### PR TITLE
Local build

### PR DESCRIPTION
**Pull request by BuDDi**

### CODE REVIEW

File: infra.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        networks:
+            - inner
+        volumes:
+            - db-data:/var/lib/mysql
+        restart: always
+        environment:
+            MYSQL_ROOT_PASSWORD_FILE: /run/secrets/db_root_password
+            MYSQL_DATABASE: openfire
+            MYSQL_USER: openfire
+            MYSQL_PASSWORD: testpass
```
Comment (nsobadzhiev) [inline infra.yml:14]: You hardcoded the password here? Then there's probably no point in having the password.txt files?

File: build/docker/entrypoint_local.sh [hunk 2]
```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```
Comment (nsobadzhiev) [inline build/docker/entrypoint_local.sh:1]: Can't you use the normal entrypoint script - how is this one different?

File: README.md [hunk 3]
```diff
@@ -93,6 +97,44 @@ Add your private key to the SSH Agent:
...
+COPY --from=packager /usr/src/plugins/openfire-avatar-upload-plugin/target/avatarupload-0.0.1-SNAPSHOT.jar \
+    /usr/src/plugins/openfire-voice-plugin/target/voice-0.0.11-SNAPSHOT.jar \
+    /usr/src/plugins/openfire-apns/target/openfire-apns.jar \
+    # add new plugins here
+    /usr/src/plugins/openfire-hazelcast-plugin/target/hazelcast-2.4.2-SNAPSHOT.jar ./plugins/
+```
+
+To create the container run this command:
+```
+docker run --env DB_USER=openfire --env DB_PASS=testpass --env DB_URL=host.docker.internal --env DB_NAME=openfire --env AWS_REGION=eu-central-1 -p 5005:5005 -p 3478:3478 -p 3479:3479 -p 5222:5222 -p 5223:5223 -p 5229:5229 -p 5275:5276 -p 5276:5276 -p 5262:5262 -p 5263:5263 -p 5701:5701 -p 7070:7070 -p 7443:7443 -p 7777:7777 -p 9090:9090 -p 9091:9091 --name openfire openfire:latest
```
Comment (nsobadzhiev) [inline README.md:133]: You definitely don't need all these ports (at least not in 95% of the Openfire cases). But it's nice to have them here anyway - it doesn't hurt.

File: build/docker/entrypoint_local.sh (hunk 2 above)
Comment (BuDDi) [reply build/docker/entrypoint_local.sh:1]: it sets remote debugging when starting the JVM

**Review by nsobadzhiev — APPROVED** [review]

**Pull request merged**